### ISSUE TITLE
ModuleNotFoundError: No module named 'pygments.lexers.special'

### ISSUE THREAD

**Issue by mosesontheweb** [sourcery-ai/sourcery#327]
### Checklist

- [x] I have searched the [Sourcery documentation](https://docs.sourcery.ai/) for the issue, and found nothing
- [x] I have checked there are no open [bugs](https://github.com/sourcery-ai/sourcery/issues?q=label%3Abug) referencing the same bug or problem

### Description

During reviewing Sourcery reports a problem by raising a ModuleNotFoundError. I tried to review several single files, this error is not raised for every file, only for some of them, but I cannot tell the reason why it works for some and some not.
Command used: `sourcery review --verbose path/to/file.py`

### Code snippet that reproduces issue

```python
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/__main__.py", line 424, in <module>
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/click/core.py", line 1130, in __call__
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/click/core.py", line 1055, in main
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/click/core.py", line 1657, in invoke
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/click/core.py", line 1404, in invoke
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/click/core.py", line 760, in invoke
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/__main__.py", line 390, in review
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/binary/command_line_interface.py", line 246, in review
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/binary/command_line_interface.py", line 381, in _review
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/binary/command_line_interface.py", line 708, in _review_one
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/binary/command_line_interface.py", line 747, in _print_review_output
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/binary/command_line_interface.py", line 772, in _print_verbose_result
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/rich/console.py", line 1694, in print
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/rich/console.py", line 1326, in render
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/rich/markdown.py", line 523, in __rich_console__
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/rich/console.py", line 1330, in render
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/rich/console.py", line 1326, in render
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/rich/syntax.py", line 609, in __rich_console__
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/rich/segment.py", line 668, in __init__
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/rich/syntax.py", line 637, in _get_syntax
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/rich/syntax.py", line 470, in highlight
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/rich/syntax.py", line 433, in lexer
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/pygments/lexers/__init__.py", line 106, in get_lexer_by_name
  File "/home/user/myproject/.venv/lib/python3.6/site-packages/sourcery/pygments/lexers/__init__.py", line 34, in _load_lexers
ModuleNotFoundError: No module named 'pygments.lexers.special'
```

### Debug Information

IDE Version: 

Sourcery Version: 

```
sourcery, version 1.0.5

Verifying user config /home/user/.config/sourcery/sourcery.yaml - success
Verifying server availability https://api.sourcery.ai/health - success
Verifying refactoring functionality - success
Verifying local database - success
```
Pygments               2.14.0

Operating system and Version:

Ubuntu 20.04.5 LTS (Focal Fossa)

**Comment by ruancomelli** [comment]
Hello @mosesontheweb, thank you for reporting this issue!

I was able to reproduce it and am already working on a fix.

**Comment by Hellebore** [comment]
Fixed with 1.0.6.
